src/components/Banner/SearchBar/index.jsx

```diff
@@ -0,0 +1,102 @@
+import styles from "./searchBar.module.css";
+import { InputAdornment, TextField } from "@mui/material";
+import { Search } from "@mui/icons-material";
+import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
+import RemoveIcon from "@mui/icons-material/Remove";
+import AddIcon from "@mui/icons-material/Add";
+
+const StyledTextField = ({
+  placeholder,
+  icon,
+  endAdornmentClass,
+  placeholderAlign,
+}) => {
+  return (
+    <TextField
+      variant="outlined"
+      placeholder={placeholder}
+      InputProps={{
+        startAdornment: (
+          <InputAdornment position="start">{icon}</InputAdornment>
+        ),
+        endAdornment: (
+          <InputAdornment position="end">
+            <AddIcon className={endAdornmentClass} />
+          </InputAdornment>
+        ),
+      }}
+      sx={{
```

Comment:
Please use styles modules. Never style components in the template. If styles modules are not working you can use styled components instead. Please either choose styled components (recommended) or styles modules and stick to the choice for all the styles in the project.

Reply:
done